frontend/lit/components/Reference.js

```diff
@@ -155,6 +155,7 @@ class Reference extends Component {
                 {data.abstract ? (
                     <div
                         className="abstracts resize-y p-2"
+                        style={data.abstract.length > 1500 ? {height: "45vh"} : null}
```

Comment:
that's nice - so it only sets a bound if it's a long abstract?

Reply:
yeah that way longer abstracts aren't overwhelming on page load, but users can still expand to show the full abstract if they wish. and there's no weird oversized box with shorter abstracts